packages/connectivity/connectivity/test/connectivity_test.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
+// @dart = 2.9
```

Comment:
connectivity hasn't moved to nnbd yet, my migration PR will handle this. Do you want to maybe revert it for now?

Reply:
`@dart = 2.9` is non-null safety.  `@dart = 2.10` enables null safety.